GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1241,6 +1241,13 @@ private void OnGridViewKeyUp(object sender, KeyEventArgs e)
             {
                 return;
             }
+			
+			// Check modifier detail if pressed.
```

Comment:
This comment does not carry any real value, please remove

Reply:
Done